src/components/frame/v5/pages/ActivityPage/hooks.tsx

```diff
@@ -1,17 +1,69 @@
 import clsx from 'clsx';
 import React from 'react';
+import { subDays, startOfDay } from 'date-fns';
 
+import { notNull } from '~utils/arrays';
+import {
+  useGetActiveMotionsQuery,
+  useGetTotalColonyActionsQuery,
+  useGetTotalColonyDomainActionsQuery,
+} from '~gql';
 import { formatText } from '~utils/intl';
 import { WidthBoxItem } from '~v5/common/WidgetBoxList/types';
+import { useColonyContext } from '~hooks';
+
+const getThirtyDaysAgoIso = () => {
+  const thirtyDaysAgo = subDays(new Date(), 30);
+  const midnightThirtyDaysAgo = startOfDay(thirtyDaysAgo);
+  return midnightThirtyDaysAgo.toISOString();
+};
 
 export const useActivityFeedWidgets = (): WidthBoxItem[] => {
+  const { colony } = useColonyContext();
+  const { domains, colonyAddress = '' } = colony ?? {};
+  const { data: motionData } = useGetActiveMotionsQuery({
+    variables: {
+      colonyId: colonyAddress ?? '',
```

Comment:
I guess we don't need ?? if we have a fallback here

Reply:
very good point lazy copy and paste strikes again :)